purchases/src/main/java/com/revenuecat/purchases/amazon/AmazonBackend.kt

```diff
@@ -6,12 +6,12 @@ import com.revenuecat.purchases.common.networking.Endpoint
 import org.json.JSONObject
 
 /** @suppress */
-typealias PostAmazonReceiptCallback = Pair<(response: JSONObject) -> Unit, (PurchasesError) -> Unit>
+internal typealias PostAmazonReceiptCallback = Pair<(response: JSONObject) -> Unit, (PurchasesError) -> Unit>
```

Comment:
Do all these `internal` changes make this a breaking change? Since that we were overexposing things in the Amazon module 😬

Reply:
oh, that's actually a good point, I didn't think people are directly depending on amazon. I will revert this visibility changes then